### ISSUE TITLE
[Suggestion]: Release the new name tag interface as separate mod

### ISSUE THREAD

**Issue by rafaelkai** [Fuzss/easy-anvils#51]
### Mod Loader

Forge

### Minecraft Version

1.20.1

### Suggestion

For people that don't want all of the anvil's tweaks present in the mod, I think some of them would appreaciate being able to rename a name tag without an anvil, so it would be neat to release that part as a standalone mod. Cheers!

**Comment by Fuzss** [comment]
We have this, but I don’t know what the chances for a Forge port anytime soon are. 
https://legacy.curseforge.com/minecraft/mc-mods/convenient-name-tags

**Comment by rafaelkai** [comment]
Yeah, I doubt that mod is going to get ported to forge anytime soon. I can't seem to find any mod adding this feature for forge 1.20.1 so that's why I made the suggestion.

**Comment by Fuzss** [comment]
It's finally been done: https://modrinth.com/mod/name-tag-upgrade
